### PR TITLE
Added codegen_lib

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by dgquintas** [comment]
Ping. We need this imported ASAP.

@ctiller @yang-g

File: build.yaml [hunk 1]
```diff
@@ -674,6 +689,13 @@ libs:
   - grpc
   - gpr_test_util
   - gpr
+- name: codegen_lib
+  build: all
```
Comment (nicolasnoble) [inline build.yaml]: This means this becomes installable, which is probably not desirable. Private maybe ?

**Comment by dgquintas** [comment]
Comments addressed. PTAL :)

**Comment by dgquintas** [comment]
This PR is ready to be re-reviewed. Note the addition of the new `codegen_test` target.

**Comment by ctiller** [comment]
Sanity is failing

**Comment by dgquintas** [comment]
It needs build system changes to allow for transitive dependencies.

**Comment by nicolasnoble** [comment]
8m3o4

**Pull request merged**